src/components/NavigationFolder.vue

```diff
@@ -263,6 +278,24 @@ export default {
 				.catch((error) => logger.error(`could not mark folder ${this.folder.id} as read`, {error}))
 				.then(() => (this.loadingMarkAsRead = false))
 		},
+		subscribeFolder(subscribed) {
+			this.changeSubscription = true
+			this.$store
+				.dispatch('subscribeFolder', {
+					accountId: this.account.id,
+					folderId: this.folder.id,
+					subscribed: subscribed,
+				})
+				.then(() => {
+					this.changeSubscription = false
+					logger.info('folder subscribed: ' + subscribed)
+				})
+				.catch((error) => {
+					logger.error('could not update subscription', {error})
+					this.changeSubscription = false
+					throw error
+				})
```

Comment:
this is fine as it is. You might improve this code a bit with `async` and `await`. Then you can do a `try`, `catch` and `finally`, where you do the cleanup step `this.changeSubscription = false` in the `finally block` to remove the duplication.

You may leave the code as it is as well :)

Reply:
Ok, something to learn for me. Will change that.